packages/jsii-rosetta/lib/translate.ts

```diff
@@ -145,14 +145,19 @@ export class SnippetTranslator {
     this.visibleSpans = calculateVisibleSpans(source);
 
     // This makes it about 5x slower, so only do it on demand
-    if (options.includeCompilerDiagnostics) {
+    if (options.includeCompilerDiagnostics || snippet.strict) {
```

Comment:
I thought `strict` implied `includeCompilerDiagnostics`? So this OR is unnecessary?

Reply:
`yargs` gives me no way to encode the `implies` relationship. Their "implies" feature is more like "if you set X, you MUST also set anything implied by X", which is not what I want here...

And then, if I go "manually" mixup the args right next to the entry point, then I also need to figure out all the aliases for that, possibly for a variety of different case translations... I don't quite fancy this.